src/licensedcode/data/rules/other-permissive_owl_10.RULE

```diff
@@ -1,6 +1,8 @@
 ---
 license_expression: other-permissive
```

Comment:
@DennisClark we have lots of versions of the Open works License here as rules, since we added the `owl-0.9.4.LICENSE` above, should we also add all these rules as seperate licenses as they are different versions?
See details below:

* other-permissive_owl_1.RULE: version 0.4
* other-permissive_owl_2.RULE: version 0.5
* other-permissive_owl_3.RULE: without any version, which license should this be a rule of
* other-permissive_owl_4.RULE: version 0.9.2
* other-permissive_owl_5.RULE: version 0.9.1
* other-permissive_owl_6.RULE: version 0.9
* other-permissive_owl_7.RULE: version 0.8
* other-permissive_owl_8.RULE: version 0.7
* other-permissive_owl_9.RULE: version 0.6
* other-permissive_owl_10.RULE: deprecated and added as the `owl-0.9.4.LICENSE`
* other-permissive_owl_11.RULE: version 0.9.3

Reply:
I don't think they are used enough to merit adding as new licenses